wiki/eiffel/setup.md

```diff
@@ -51,7 +51,10 @@ Linux | 8 GB RAM |16 GB RAM or more |	~11 GB of RAM will be use under the load.
 
 #### Docker requirements
  - Docker 18.06 CE or newer
- - Docker-Compose 1.22 or newer
+ - In Linux Systems
+     - Use Docker-compose version <= 1.29.2
```

Comment:
Or update the docker-compose yaml files to be compliant with the newer docker-compose versions.
I have not used docker-compose for long time and not followed what have changed, but feel free to update so easy2use so it works with the newer Docker-compose version.

Reply:
Ok, so docker-compose have removed the support of providing Java properties via env section.
Spring based java Eiffel components, which all are I think, support underscore in Java properties, so might be possible to replace all dots with underscore in all Java properties.
I think this could be solved in a future tasks.